### PR TITLE
Improve documentation

### LINKED ISSUE #1295: Missing Alpine linux in the distribution list
# Documentation issue

https://www.getzola.org/documentation/getting-started/installation/
https://github.com/getzola/zola/blob/master/docs/content/documentation/getting-started/installation.md

:warning: Only available on edge: https://pkgs.alpinelinux.org/packages?name=zola

It moved one month ago from testing into community: https://gitlab.alpinelinux.org/alpine/aports/-/commit/a8d88dedc8692fbe7b3b22a46e4309a4e38a7f2d

## Summary
Missing Alpine linux in the distribution list

## Proposed solution
Add alpine linux and warn that it is only on edge for now

## Details

Test install command line using docker: `docker run --rm -it alpine apk add zola --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community`

Command to document `apk add zola --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community`

### Proposal

image

-------

Other question: no docker version ? Need contributions ?

### PR DESCRIPTION
**Pull request by williamdes**
Improve documentation:
- HTTP to HTTPS for some links
- use sh where is is possible
- document alpine version (Fixes: #1295)
- typo fixes
- add my website
- Some version updates
- Some example updates
- Removed some (but a lot are remaning) useless trailing whitespaces
- Typo fixes
- Updated more example versions (abf6ec71e2518230281bb478e591d5f75e368032)

### CODE REVIEW

File: docs/content/documentation/content/search.md [hunk 1]
```diff
@@ -22,5 +22,5 @@ the JavaScript/CSS code to do an actual search and display results. You can look
 implements it to get an idea: [search.js](https://github.com/getzola/zola/tree/master/docs/static/search.js).
 
 ## Configuring the search index
-In some cases, the default indexing strategy is not suitable. You can customise which fields to include and whether
+In some cases, the default indexing strategy is not suitable. You can customize which fields to include and whether
```
Comment (Keats) [inline docs/content/documentation/content/search.md:25]: ah the old US English vs rest of the world English ;) I prefer customise but I wouldn't say this needs to be changed

File: docs/content/documentation/deployment/github-pages.md [hunk 2]
```diff
@@ -33,19 +33,19 @@ on: push
...
-      env:
-        # Target branch
-        PAGES_BRANCH: gh-pages
-        # Provide personal access token
-        TOKEN: ${{ secrets.TOKEN }}
+      # Checkout
+      - uses: actions/checkout@v2
+      # Build & deploy
+      - name: Deploy to gh-pages branch
+        uses: shalzz/zola-deploy-action@v0.12.0
```
Comment (Keats) [inline docs/content/documentation/deployment/github-pages.md]: let's put 0.13 here as well

**Review by Keats — APPROVED** [review]

File: docs/content/documentation/deployment/github-pages.md (hunk 2 above)
Comment (williamdes) [reply docs/content/documentation/deployment/github-pages.md]: Done in abf6ec71e2518230281bb478e591d5f75e368032

**Comment by williamdes** [comment]
I also fixed all broken links (fa9a4008be9a5e8631259c7bf985339c0b8bd41d) found by [blc](https://www.npmjs.com/package/broken-link-checker) `~/.yarn/bin/blc -r -v https://www.getzola.org/documentation/getting-started/overview/`
I had to add a missing screenshot

PS: Some links (3) used the .com ltd but it seems not to work, I changed them to .org

**Comment by Keats** [comment]
The theme folder is auto-generated so it will be overwritten the next time a theme is updated. Thanks for the rest though!

**Comment by williamdes** [comment]
> The theme folder is auto-generated so it will be overwritten the next time a theme is updated. Thanks for the rest though!

Oh okay, so I need to open PRs on each theme I updated ?

**Comment by Keats** [comment]
> Oh okay, so I need to open PRs on each theme I updated ?

If you want. I try to do some PRs to the theme when I can but it would be helpful to have more hands/eyes

**Pull request merged**